src/pages/welcome.html

```diff
@@ -19,15 +19,17 @@
     <main class="section document">
       <div class="row">
         <div class="document-content">
-          <h1 class="document-headline ff-Met">Welcome to the Firefox Private Network Beta</h1>
-          <p class="lead">Thanks for participating in this limited-time, free beta of Firefox Private Network. You can use this lightweight browser extension to enhance your privacy and security on Firefox for desktop. Use it to:</p>
+          <h1 class="document-headline ff-Met">Welcome to the Firefox Private Network</h1>
```

Comment:
I think this should be "Welcome to the Firefox Private Network Beta" again, per @betsymi's https://github.com/mozilla/secure-proxy/issues/797#issuecomment-557530806